src/fix/main.c

```diff
@@ -689,36 +704,56 @@ static void processFile(int input, int output, char const *name, off_t fileSize)
 		}
 	}
 
-	if (title)
+	if (title) {
+		nonZeroWarn(rom0, 0x134, titleLen, "Title");
 		memcpy(&rom0[0x134], title, titleLen);
+	}
 
-	if (gameID)
+	if (gameID){
+		nonZeroWarn(rom0, 0x13f, gameIDLen, "Manufacturer Code");
 		memcpy(&rom0[0x13f], gameID, gameIDLen);
+	}
 
-	if (model != DMG)
+	if (model != DMG) {
+		nonZeroWarn(rom0, 0x143, 1, "CGB Flag");
 		rom0[0x143] = model == BOTH ? 0x80 : 0xc0;
+	}
 
-	if (newLicensee)
+	if (newLicensee) {
+		nonZeroWarn(rom0, 0x144, newLicenseeLen, "New Licensee Code");
 		memcpy(&rom0[0x144], newLicensee, newLicenseeLen);
+	}
 
-	if (sgb)
+	if (sgb) {
+		nonZeroWarn(rom0, 0x146, 1, "SGB Flag");
 		rom0[0x146] = 0x03;
+	}
 
 	// If a valid MBC was specified...
-	if (cartridgeType < MBC_NONE)
+	if (cartridgeType < MBC_NONE) {
+		nonZeroWarn(rom0, 0x147, 1, "Cartridge Type");
 		rom0[0x147] = cartridgeType;
+	}
 
-	if (ramSize != UNSPECIFIED)
+	if (ramSize != UNSPECIFIED) {
+		nonZeroWarn(rom0, 0x149, 1, "RAM Size");
 		rom0[0x149] = ramSize;
+	}
 
-	if (!japanese)
+	if (!japanese) {
+		nonZeroWarn(rom0, 0x14a, 1, "Destination Code");
 		rom0[0x14a] = 0x01;
+	}
 
-	if (oldLicensee != UNSPECIFIED)
+	if (oldLicensee != UNSPECIFIED) {
+		nonZeroWarn(rom0, 0x14b, 1, "Old Licensee Code");
 		rom0[0x14b] = oldLicensee;
+	}
 
-	if (romVersion != UNSPECIFIED)
+	if (romVersion != UNSPECIFIED) {
+		nonZeroWarn(rom0, 0x14c, 1, "Mask ROM Version number");
```

Comment:
Either capitalize `number` or (my personal preference) make all the area names lowercase (except for terms like `Nintendo` or `ROM` where capitals are necessary).